src/m3u8/parse-stream.js

```diff
@@ -130,7 +130,7 @@ export default class ParseStream extends Stream {
         tagType: 'inf'
       };
       if (match[1]) {
-        event.duration = parseFloat(match[1]);
+        event.duration = Math.max(0.01, parseFloat(match[1]));
```

Comment:
When we've defaulted m3u8 attributes elsewhere, we've done it in the [parser and emitted an info event](https://github.com/videojs/videojs-contrib-hls/blob/23f4b359fd3df68d6d13648c0e2790aeb73b0453/src/m3u8/parser.js#L70)